aspnet-core/xml/ns-Microsoft.AspNetCore.StaticFiles.Infrastructure.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.StaticFiles.Infrastructure">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains shared option types used by static files middlewares.</summary>
```

Comment:
Taking a quick look at the types in this namespace and how they are used, I'm wondering we even expose these types publicly at all @Tratcher?